capstone_project/src/main/webapp/styles/maps_style.css

```diff
@@ -1,19 +1,43 @@
-#content {
-  width: 100%;
-  height: 100%;
-  margin: 0;
-  font: 15px 'Quicksand', sans-serif;
+body {
```

Comment:
Can you sort the declarations (i.e color, font-family, margin, etc) in [alphabetical order](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)?

Reply:
Also nit: For consistency throughout the file, use hex code instead of words for color declaration. black: #000, white: #fff

Reply:
sounds good, just fixed it! @Leoism